modules/v1/workspaces/workspaceBehaviours/TimelineUrlBehaviour.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+
+namespace app\modules\v1\workspaces\workspaceBehaviours;
+
+use app\modules\v1\workspaces\models\TimelinePost;
+use yii\base\Behavior;
+use yii\db\ActiveRecord;
+
+class TimelineUrlBehaviour extends Behavior
+{
+    public $timelinePost;
+
+    public function events()
+    {
+        return [
+            ActiveRecord::EVENT_AFTER_INSERT => 'afterInsert',
+            ActiveRecord::EVENT_AFTER_UPDATE => 'afterInsert',
+        ];
+    }
+
+    public function afterInsert() {
+        if(strlen($this->timelinePost->description !== 0)) {
+            $description = $this->timelinePost->description;
+            $description = strip_tags(preg_replace("((<\/)\w+(>))", "\n", $description));
```

Comment:
Can you explain what this does?

Reply:
Basically, description in db is kept with html format(<p>description</p>). I strip tags from description and apply link regex on "clean description"